src/layout/content/content.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+
+export interface MainContentProps {
+  readonly children: React.ReactNode
+  readonly topBarSelector?: string
+  readonly topBarHeight?: string
```

Comment:
Can we derive the height from the topBarSelector above ?
If we need to keep this prop, can we document what expected in the string param (I guess something like '24px')

Reply:
We need this prop for when extracting the CSS at build (for SSR for example) as the topbar isn't mounted at this moment.
I'll had a comment :)